app/web/src/components/FuncEditor/ManagementDetails.vue

```diff
@@ -35,6 +98,66 @@ const binding = computed(() => {
   return binding;
 });
 
+const schemaOptions = computed(() => {
+  const uninstalled: Option[] = assetStore.uninstalledVariantList.map(
+    (unin) => ({
+      label: unin.displayName ?? unin.schemaName,
+      value: unin.schemaId,
+    }),
+  );
+
+  const installed = assetStore.variantList.map((inst) => ({
+    label: inst.displayName ?? inst.schemaName,
+    value: inst.schemaId,
+  }));
+
+  const currentManagedSchemas = binding.value?.managedSchemas ?? [];
+
+  const allOptions = uninstalled.concat(installed);
+  allOptions.forEach((option) => {
+    const schemaId = option.value.toString();
+    schemaNameMap.value[schemaId] = option.label;
+  });
+
+  const filteredOptions = _.uniq(
+    allOptions.filter(
+      (opt) =>
+        typeof opt.value === "string" &&
+        opt.value !== managerSchemaId.value &&
+        !currentManagedSchemas.includes(opt.value),
+    ),
+  );
+  filteredOptions.sort((a, b) => a.label.localeCompare(b.label));
+
+  return filteredOptions;
+});
+
+const addSchema = async () => {
+  if (binding.value) {
+    const updated_binding = _.clone(binding.value);
```

Comment:
BTW, I prefer using `structuredClone` (yay ES6) rather than lodash. It is both faster and more thorough/correct.